application/consumer-activemq-quarkus/src/test/java/webapp/tier/service/ActiveMqSubscribeServiceTest.java

```diff
@@ -1,27 +1,56 @@
 package webapp.tier.service;
 
-import static org.junit.jupiter.api.Assertions.*;
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assertions.fail;
 
 import javax.inject.Inject;
+import javax.jms.ConnectionFactory;
+import javax.jms.JMSContext;
+import javax.jms.Session;
 
+import org.eclipse.microprofile.config.inject.ConfigProperty;
 import org.junit.jupiter.api.Test;
 
 import io.quarkus.artemis.test.ArtemisTestResource;
 import io.quarkus.test.common.QuarkusTestResource;
 import io.quarkus.test.junit.QuarkusTest;
+import webapp.tier.bean.MsgBean;
+import webapp.tier.util.CreateId;
+import webapp.tier.util.MsgUtils;
 
 @QuarkusTest
 @QuarkusTestResource(ArtemisTestResource.class)
 class ActiveMqSubscribeServiceTest {
 
 	@Inject
-	ActiveMqSubscribeService svc;
+	ActiveMqSubscribeService subsvc;
+
+	@Inject
+	ActiveMqService svc;
+
+	@Inject
+	ConnectionFactory connectionFactory;
+
+	@ConfigProperty(name = "common.message")
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/yurake/k8s-3tier-webapp/pullRequest?prid=6157602)